scripts/cl/one_for_zero.py

```diff
@@ -34,8 +35,29 @@ def calc_test_case(liquidity: sp.Float, sqrt_price_current: sp.Float, token_in:
 
     return sqrt_price_next, token_out, fee_amount_per_share
 
-def calc_test_case_with_next_sqrt_price(liquidity: sp.Float, sqrt_price_current: sp.Float, sqrt_price_next: sp.Float, swap_fee: sp.Float) -> Tuple[sp.Float, sp.Float, sp.Float]:
+def calc_test_case_in_given_out(liquidity: sp.Float, sqrt_price_current: sp.Float, token_in: sp.Float, swap_fee: sp.Float) -> Tuple[sp.Float, sp.Float, sp.Float]:
+    """ Computes and prints all one for zero test case parameters when swapping for in given out.
+    Next sqrt price is computed from the given parameters.
+
+    Returns the next square root price, token in and fee amount per share.
+    """
+    sqrt_price_next = get_next_sqrt_price(liquidity, sqrt_price_current, token_in)
+    price_next = sp.Pow(sqrt_price_next, 2)
+    token_in = get_token_out(liquidity, sqrt_price_current, sqrt_price_next)
```

Comment:
why do we assign `token_in` to a func named `get_token_out`?

Reply:
Similar idea as in this discussion with the Go tests:
https://github.com/osmosis-labs/osmosis/pull/4131#discussion_r1091867438

I can make a wrapper function `get_token_in_swap_in_given_out` around `get_token_out` and call `get_token_in_swap_in_given_out` here to make it clearer

Reply:
Done: [c3d6116](https://github.com/osmosis-labs/osmosis/pull/4131/commits/c3d61168b29cee677edca8ab38cd6b783c08a409)